### DISCUSSION TITLE
Github page doesn't update

### DISCUSSION THREAD

**Discussion by marcsi19** [community/community#22988] [Other Feature Feedback, Questions, & Ideas]
<p>Hi!</p>
<p>I have a problem with my website Github page. I updated today and the page won’t update. There was a high risk vulnerability (lodash.template needed an update) and I opted for auto fix by Github. Once it was updated, I pulled and pushed with the fixes. The risk is resolved but my page hasn’t updated yet. It was about 2 hours ago. </p>

**Comment by marcsi19** [comment, marked as answer]
<p>npm run deploy</p>
<p>Don’t forget this step</p>

**Comment by wqw547243068** [comment]
<p>I have encountered similar problems with you, my page won’t update with markdown file.<br>
I use jekyll to build my website, it runs well locally, however, this GitHub page starts to stay in the old version from last week.<br>
The error information in the <a>setting page</a> shows as the following, and I don’t know why</p>
<pre><code class="lang-auto">The page build failed for the `master` branch with the following error:  
Page build failed. For more information, see [https://help.github.com/en/articles/troubleshooting-github-pages-builds](https://help.github.com/en/articles/troubleshooting-github-pages-builds).  
For information on troubleshooting Jekyll see:  
[https://help.github.com/articles/troubleshooting-jekyll-builds](https://help.github.com/articles/troubleshooting-jekyll-builds)  
If you have any questions you can contact us by replying to this email.  
</code></pre>

**Comment by dlintin** [comment]
<p>facing the same problem</p>

**Comment by ghost** [comment]
<p>Is there any solution for this issue?</p>

**Comment by britzl** [comment]
<p>I also encountered this problem yesterday. My page stopped generating on changes and didin’t go back to normal even after I reverted the most recent change.</p>

**Comment by britzl** [comment]
<p>There was actually a problem with some pre-formatted text that Jekyll wasn’t able to detect language for. When I updated the GitHub Pages gem and rebuilt the site locally (non-incremental) I got:</p>
<p>Conversion error: Jekyll::Converters::Markdown encountered an error while converting ‘_manuals/extensions-manifest-merge-tool.md’:<br>
Rouge::Guesser::Ambiguous<br>
jekyll 3.8.5 | Error: Ambiguous guess: can’t decide between [“mason”, “xml”]</p>
<p>It’s weird though that GitHub couldn’t provide a better error message.</p>

**Comment by jamescaan20** [comment]
<p>Any video to fix this issue. I am also facing same issue.</p>

**Comment by abdulsamad786** [comment]
I am having the same issue
